tox.ini

```diff
@@ -1,13 +1,13 @@
 [tox]
-envlist = flake8, py{27,35,36,37}, coverage, mkdocs
+envlist = flake8, py{36,37,38,39,310}, coverage, mkdocs
 
 [testenv]
 extras = test
 sitepackages = False
 commands = pytest -v {posargs}
 
 [testenv:flake8]
-basepython = python3.6
+basepython = python3.8
```

Comment:
Better to use the latest python version instead of something in the middle? we want the syntax style and features from the latest and not in the middle

Reply:
Is this comment still relevant after my latest commits?

Reply:
This is the only remaining comment so feel free to give it a thumbs up or down and then we are good to go.